# Aouizerat_Gelman_et_al_2019
Analysis for the manuscript "Eukaryotic adaptation to years-long starvation resembles that of bacteria"  
  
[![DOI](https://zenodo.org/badge/167777256.svg)](https://zenodo.org/badge/latestdoi/167777256)
  
**Genome assebly and ploidy reconstruction**  
[Tabulate raw reads](https://nbviewer.jupyter.org/github/DSASC/Aouizerat_Gelman_et_al_2019/blob/master/script.ipynb#1.-Tabulate-fastq-files)  
[Trim sequence reads](https://nbviewer.jupyter.org/github/DSASC/Aouizerat_Gelman_et_al_2019/blob/master/script.ipynb#2.-Trim-reads)  
[Inspect sequence data quality](https://nbviewer.jupyter.org/github/DSASC/Aouizerat_Gelman_et_al_2019/blob/master/script.ipynb#3.-Fastqc)  
[Denovo assembly with Celera](https://nbviewer.jupyter.org/github/DSASC/Aouizerat_Gelman_et_al_2019/blob/master/script.ipynb#4.1-Celera)  
[Denovo assembly with Platanus](https://nbviewer.jupyter.org/github/DSASC/Aouizerat_Gelman_et_al_2019/blob/master/script.ipynb#4.2-Platanus)  
[Make blobplots](https://nbviewer.jupyter.org/github/DSASC/Aouizerat_Gelman_et_al_2019/blob/master/script.ipynb#5.-Plot-blobs)  
[Select Celera results based on blobplots](https://nbviewer.jupyter.org/github/DSASC/Aouizerat_Gelman_et_al_2019/blob/master/script.ipynb#6.-Denovo-assembly-conclusions-from-blobtools:)  
[Assembly statistics](https://nbviewer.jupyter.org/github/DSASC/Aouizerat_Gelman_et_al_2019/blob/master/script.ipynb#7.-Assembly-statistics)  
[Determine ploidy with Smudgeplot](https://nbviewer.jupyter.org/github/DSASC/Aouizerat_Gelman_et_al_2019/blob/master/script.ipynb#8.-Ploidy---smudgeplots)  
**Chromosome coverage in 404 and 402 compared to SefAle**  
[Get coverage of the wild type genome by 404,424 and SelAle reads](https://nbviewer.jupyter.org/github/DSASC/Aouizerat_Gelman_et_al_2019/blob/master/script.ipynb#10.-Map-reads-to-reference-genome)  
[Plot coverage of each chromosome to study the extand of ploidy change](https://nbviewer.jupyter.org/github/DSASC/Aouizerat_Gelman_et_al_2019/blob/master/script.ipynb#10.4-Plot-coverage)  
[SNP calling](https://nbviewer.jupyter.org/github/DSASC/Aouizerat_Gelman_et_al_2019/blob/master/script.ipynb#11.-SNP-calling)  
[Functional annotation of wild type genome](https://nbviewer.jupyter.org/github/DSASC/Aouizerat_Gelman_et_al_2019/blob/master/script.ipynb#12.-Functional-annotation-of-reference-genome-genes)  
[Get TOR1 sequences in each sample](https://nbviewer.jupyter.org/github/DSASC/Aouizerat_Gelman_et_al_2019/blob/master/script.ipynb#13.-Consensus-sequences-for-the-TOR1-gene)  
**Gain and loss of heterozygocity, Shared and unique polymorphism, in 404 and 424**  
[Map 404 and 424 to SefAle genome](https://nbviewer.jupyter.org/github/DSASC/Aouizerat_Gelman_et_al_2019/blob/master/script.ipynb#14.-Map-404-and-402-to-SefAle)  
[SNP call](https://nbviewer.jupyter.org/github/DSASC/Aouizerat_Gelman_et_al_2019/blob/master/script.ipynb#15.-SNPs-in-SefAle,-404-and-424-compared-to-SefAle)  
[Functional annotation of SefAle](https://nbviewer.jupyter.org/github/DSASC/Aouizerat_Gelman_et_al_2019/blob/master/script.ipynb#16.-identify-genes-in-SefAle)  
[Summarize polymorphisms](https://nbviewer.jupyter.org/github/DSASC/Aouizerat_Gelman_et_al_2019/blob/master/script.ipynb#19.-Summarize-polymorphism-in-404,-424-compared-to-SefAle)  
[Synonymous and non synonymouse point mutations in descendant strains](https://nbviewer.jupyter.org/github/DSASC/Aouizerat_Gelman_et_al_2019/blob/master/script.ipynb#21.-Detect-synonymous-and-non-synonymou- mutation)  
[Venn diagrams of share and unique genes genes with point mutation](https://nbviewer.jupyter.org/github/DSASC/Aouizerat_Gelman_et_al_2019/blob/master/script.ipynb#22.-Venn-diagrams-of-genes-with-point-mutations,-homozygote-in-SefAle-and-homozygote-in-424,-404)
[Gain and Loss of heterozygocity](https://nbviewer.jupyter.org/github/DSASC/Aouizerat_Gelman_et_al_2019/blob/master/script.ipynb#23.-Summarize-gain-and-loss-of-heterozygocity)
[Summarize polymorphism types (for Figure-4C)](#24.-Summarize-polymorphism-types-for-Figure-4C)